ccm_web/server/src/api/dtos/api.create.sections.dto.ts

```diff
@@ -1,13 +1,19 @@
-import { IsNotEmpty, MaxLength } from 'class-validator'
+import {
+  ArrayMaxSize,
+  ArrayMinSize,
+  IsArray
+} from 'class-validator'
 import { ApiProperty } from '@nestjs/swagger'
 
 export class CreateSectionsDto {
   @ApiProperty({ type: [String] })
-  @IsNotEmpty({ each: true })
-  @MaxLength(250, { each: true })
```

Comment:
Why did you remove these? I think we still want to enforce that each section name string is `> 0` and `< 256` characters, because otherwise users will hit up on Canvas validation errors.

Reply:
You are right. I added those checks back.